.travis.yml

```diff
@@ -27,24 +39,28 @@ before_install:
 install:
   # Upgrade pip and setuptools. Mock has issues with the default version of
   # setuptools
-  - |
-    pip install --upgrade pip
-    pip install --upgrade setuptools
-    pip install $NUMPY
-    pip install $MPL
-    pip install -r requirements.txt
+  - pip install --upgrade pip
+  - pip install --upgrade setuptools
+
+  - pip install $NUMPY
+  - pip install $MPL
```

Comment:
it is something to consider, but let's not get ahead of ourselves here. One thing I do want to make sure of is that we are testing against the minimum requirements as well as the current requirements, which is what the config is currently set up to do. If we were to install numpy and matplotlib from APT, then we lose that.

(Yes, I know we aren't testing against older matplotlibs. We should probably fix that)